components/psst/common/BUILD.gn

```diff
@@ -13,11 +13,15 @@ mojom("mojom") {
 }
 
 static_library("common") {
+  public = [
+    "brave_psst_constants.h",
```

Comment:
just constants.h
also public isn't really needed here because nothing is non-public